index.html

```diff
@@ -0,0 +1,115 @@
+
+    <div class="whole-page">
+
+<!------------- S I G N  I N  B U T T O N  S E C T I O N ---------------->
+
+        <header class="signUp-header">
+
+            <div class="headerflex">
+                <div class="header-text">
+                    <span class="text-1">Already have an Intuit account?</span> <br>
+                    <span class="text-2"><a
+                            href="https://accounts.intuit.com/account-recovery.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3Dd14c5fd2-f008-4b43-a39f-e3e5e91b66ee%26adobe_mc%3DMCMID%253D63929106516021020063909284941278657742%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1600873524%26ivid%3Dd14c5fd2-f008-4b43-a39f-e3e5e91b66ee">I
+                            forgot my User ID or Password</a></span>
+                </div>
+            </div>
+
+            <button type="button" class="signUp-button"><i class="material-icons lock">lock</i><span class="SignIn-text">Sign In</span></button>
+
+        </header>
+
+<!-------------------- L O G O  A N D  I C O N S ----------------------->
+
+        <main class="main-cont">
+
+            <section class="logo-icons-cont">
+
+                <div class="logo-container">
+                    <img src="images/intuit-brand1.png" alt="intruit-logo" class="intuit-brand-logo">
+                </div>
+
+                <div class="three-icons-cont">
+                    <div>
+                        <a href="https://www.mint.com/" target="_blank"><img src="images/mint-img.png" alt="mint-icon" class="mint-brand-logo"></a>
+                    </div>
+                    <div>
+                        <a href="https://quickbooks.intuit.com/" target="_blank"><img src="images/quickbooks-img.png" alt="quickbooks-icon" class="quickbooks-brand-logo"></a>
+                    </div>
+                    <div>
+                        <a href="https://turbotax.intuit.com/" target="_blank"><img src="images/turbotax-img.png" alt="turbotax-icon" class="turbotax-brand-logo"></a>
+                    </div>
+                </div>
+
+            </section>
+
+<!------------------------------ F O R M --------------------------------->
+
+            <section class="mainForm-cont">
+
+                <span class="intuitHead">Create an Intuit account</span>
+                <span class="intuitAcount">One account for everything Intuit, including Mint.</span>
+                <span class="learn-more"><a href="https://accounts-help.intuit.com/app/intuit/1995107" target="_blank">Learn more</a></span>
+
+                <form class="form-cont">
+
+                    <span class="email">Email Address</span><br>
+                    <input type="text" class="emailInput"><br><br><br>
+                    <span class="phone">Phone (recommended)</span><br>
+                    <input type="text" class="phoneInput"><br>
```

Comment:
Please, modify your phone type into `tel`
It has an `<input>` of type `tel` and a label `Phone` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))